### ISSUE TITLE
[WRONG CODE] intra_attation function

### ISSUE THREAD

**Issue by Albert-Ma** [vanzytay/NIPS2018_DECAPROP#2]
In `tylib/lib/att_op` file, this function was called by `bidirectional_attention_connector()` with wrong parameter.

And also skip that, the dev result while in training was always the same since the loss did not decrease, no matter how many epochs, it always got the same loss, same F1, same EM.

Can you clean your code, and give a success version?

**Comment by Albert-Ma** [comment]
@vanzytay

**Comment by vanzytay** [comment]
Code is copied from main infrastructure codebase and I have not tested this repository. This is why you find many hyperparameter settings not in order. Also, the command of the model calling is likely to not be right. There is no intention to clean this repository. 

For usage I suggest actually copying the model code to your own codebase.
